### ISSUE TITLE
[Designer] Bug 37603545: [ActivityUpdate.Json]: Incorrect role as 'Button' is defined for the 'Set Due date' control on Activity Update card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7619]
### Problem Description

[Bug 37603545](https://microsoft.visualstudio.com/OS/_workitems/edit/37603545): [ActivityUpdate.Json]: Incorrect role as 'Button' is defined for the 'Set Due date' control on Activity Update card.

Test Environment:
Application: Adaptive Cards
OS version: Dev (OS Build 22499.1010)
Application Version: 3.0.2111.24001
Screen Reader: Narrator

Repro steps:
1. Launch the Adaptive Card application and enable Narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to ActivityUpdate.Json button by using Tab key and activate it.

4. ActivityUpdate adaptive card will be appeared on right sight of the pane.

5. Navigate to ActivaityUpdate card by using Tab key.

6. Navigate to "Set Due Date" button and activate it.

7. Observe the issue with screen reader.

Actual Result:
Incorrect role as 'Button' is defined for the 'Set Due date' control on Activity Update card. Also, state is not defined for 'Set Due Date' control. Screen reader announces as, "Set Due Date Button".

Expected Result:
Screen reader should announce correct role as, "Menu item" for "Set Due date" control along with the position and state information as, "Set due date menu item 1 of 2 expanded/collapsed".

Note: Same issue is also repro with the 'Set Due Date' control on 'ActivityUpdateWithLabels' card.

User Experience:
Screen reader users will get confused if screen reader will announce incorrect role information for 'Set Due Date' control or does not announce position information, Screen reader users will also not know about the exact state of 'Set Due Date' control even if it does not announce the state information.

Observation: Same issue is also repro with NVDA.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
Same issue is repro on the below environment.

**Test Environment:**
Device: iPhone 11
iOS: 15.6.1
Application: Adaptive Cards
App Version: Version 1.0(2.4.0-beta.22.9.4.1)

**Comment by vagpt** [comment]
Closing this issue as per the discussion with internal team.
